addons/io_scene_gltf/camera.py

```diff
@@ -2,7 +2,23 @@
 
 
 def create_camera(op, idx):
-    camera = op.gltf['cameras'][idx]
-    name = camera.get('name', 'cameras[%d]' % idx)
-    data = bpy.data.cameras.new(name)
-    return data
+    """
+    create a new blender camera from the gltf data
+    """
+    data = op.gltf['cameras'][idx]
+    name = data.get('name', 'cameras[%d]' % idx)
+    camera = bpy.data.cameras.new(name)
+    if data["type"] == "orthographic":
+        camera.type = "ORTHO"
+        p = data["orthographic"]
+        camera.clip_start = p["znear"]
+        camera.clip_end = p["zfar"]
+        camera.ortho_scale = max(p["xmag"], p["ymag"])
+    else:
+        camera.type = "PERSP"
+        p = data["perspective"]
+        camera.clip_start = p["znear"]
+        camera.clip_end = p["zfar"]
```

Comment:
The `zfar` property isn't required. When its not there the camera should use "infinite projection" (not sure how to do that in Blender).